diff --git a/node_modules/@prisma/client/index.js b/node_modules/@prisma/client/index.js
index 4b9542095033333b28b75968d9047f334a17a740..587593c66f7f2b5a190b9b3e157e3352eb9b8032 100644
--- a/node_modules/@prisma/client/index.js
+++ b/node_modules/@prisma/client/index.js
@@ -102,4 +102,4 @@
 'transaction': ['error'],
 };
 
-module.exports = { PrismaClient };
+module.exports = { ...require('./default'), ...require('./system') }
diff --git a/node_modules/@prisma/client/index.d.ts b/node_modules/@prisma/client/index.d.ts
index 78d7890f55562725e17135e8869c97ed5f9175d0..3d387f3b51680517596a798939c8942b083b4e31 100644
--- a/node_modules/@prisma/client/index.d.ts
+++ b/node_modules/@prisma/client/index.d.ts
@@ -2,4 +2,4 @@
 
 export { type PrismaClient } from './default'
 
-export * from './default';
+export * from './default'
+export * from './system'
